build.sbt

```diff
@@ -73,6 +73,10 @@ lazy val langmeta = crossProject
     publishableSettings,
     crossScalaVersions := List(LatestScala210, LatestScala211, LatestScala212),
     description := "Langmeta umbrella module that includes all public APIs",
+
+    // to use your system's protoc uncomment the following line
+    // PB.runProtoc in Compile := { args => Process("protoc", args).! },
```

Comment:
👍 Thanks for debugging this

Reply:
Let's make this our build requirement? Or open an issue and mention this solution there as a workaround? I'm generally nervous about leaving executable code commented out.

Reply:
I think this is specific to @MasseGuillaume's local setup, scalapb brings in its own dependencies

Reply:
Yup sbt-protoc depends on `"com.github.os72" % "protoc-jar" % "3.5.0"` this is basically an exe in a jar. This is totally disabled in a sane system like nixos.